test/CodeGenerator/AbstractInjectorTest.php

```diff
@@ -187,17 +187,9 @@ public function testConstructionWithoutContainerUsesDefaultContainer()
     public function testFactoryIsCreatedFromClassNameString()
     {
         $subject = $this->createTestSubject(function () {
-            return ['SomeClass' => StdClassFactory::class ];
+            return ['SomeClass' => StdClassFactory::class];
         });
 
-        $this->assertSame(
-            StdClassFactory::class,
-            self::readAttribute($subject, 'factories')['SomeClass'] ?? null
-        );
```

Comment:
This and the other assertion check the internal state of the class, which is considered bad practice and `readAttribute()` has been removed from phpunit (see sebastianbergmann/phpunit#3338).

Testing the result of `create()` should be sufficient.